sky/schemas/proto/servev1.proto

```diff
@@ -0,0 +1,88 @@
+syntax = "proto3";
+
+package serve.v1;
+
+service ServeService {
+    // Get status of service.
+    rpc GetServiceStatus(GetServiceStatusRequest) returns (GetServiceStatusResponse);
+    // Add version to service.
+    rpc AddVersion(AddVersionRequest) returns (AddVersionResponse);
+    // Terminate services.
+    rpc TerminateServices(TerminateServiceRequest) returns (TerminateServiceResponse);
```

Comment:
Nit for consistency:

```suggestion
    rpc TerminateServices(TerminateServicesRequest) returns (TerminateServicesResponse);
```

Reply:
This reminds me, maybe we should add a protobuf linter too in our CI. Locally I'm using https://buf.build/docs/lint/.